tests/cupyx_tests/scipy_tests/ndimage_tests/test_filters.py

```diff
@@ -238,6 +252,18 @@ def test_filter(self, xp, scp):
             testing.shaped_random((26, 3, 52, 70, 50), xp, xp.float32),
             [1, 1, 1, 1, 1], mode='nearest')
 
+    @testing.with_requires('scipy>=1.x.x')
+    @testing.numpy_cupy_allclose(atol=1e-5, rtol=1e-5)
+    def test_gaussian_filter_backend(self, xp):
```

Comment:
`TestFilter.test_filter_backend` would be a better place, (or add a new test class).